File: java/src/main/java/mx/iteso/adapter/Format.java
package mx.iteso.adapter;

import java.awt.Font;

/**
 * Format of doc.
 */
public class Format {
    /**
     * font.
     */
    private Font font;

    /**
     * define format.
     */
    public Format() {
        final int fontSize = 20;
        this.font = new Font("Arial", Font.BOLD, fontSize);
    }

    /**
     *
     * @return font of doc.
     */
    public Font getFont() {
        return font;
    }
}

File: csharp/AdapterPractice/AdapterPractice/Adapter/Format.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AdapterPractice.Adapter
{
    public class Format
    {
       
        private Font font;

       
        public Format()
        {
            int fontSize = 20;
            this.font = new Font("Arial");
        }

       
        public Font getFont()
        {
            return font;
        }
    }
}
